src/Round.js

```diff
@@ -1,43 +1,35 @@
 import Player from './Player.js';
 import Turn from './Turn.js';
 import Game from './Game.js';
+import FastMoneyTurn from './FastMoneyTurn.js';
 
 class Round {
-  constructor(game, surveys, answers) {
+  constructor(game, survey, answers) {
     this.game = game;
```

Comment:
I don’t know if I’d store the game instance as a property on the Round — if instance properties on your game are updating/changing, remember that you’re constructor of Round only runs once — so the game on your round will not update along with the instance of the current game. Instead, I would pass through your game instance to any methods that actually need them so you know you’re working with the most current version of your game when your methods are called.

Reply:
(It looks like you *are* passing game through as an argument to most of your methods, which is the way to go, but then I’m curious where/how/why you’d be using `this.game` from your round class.

Reply:
Thanks! We noticed that and removed it.